app/styles/libs/_helpers.scss

```diff
@@ -98,3 +98,7 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
 .no-wrap {
   white-space: nowrap;
 }
+
+.card-height {
+  min-height: 165px;
```

Comment:
Is this needed?

Reply:
Yes, else in dekstop view the cards will not be of equal size ( online + location events have greater height). I have removed this in mobile view so extra height is not added there